crates/backend/.gitignore

```diff
@@ -0,0 +1 @@
+generated-bindings
```

Comment:
We generate TypeScript bindings with `ts-rt` so we need to gitignore the output.

Reply:
I'm in favor of ts-rs, looks great! But after running make build and also cargo build, the generated-bindings folder and also the CompilationResult/Request.ts files don't show up anywhere. Looks like they are currently not generated?

Reply:
Oh, that's not so well documented. The generation is triggered on `cargo test` and not `cargo build`.
That's the behavior baked in `rt-ts`.

I'm working on a separate PR that documents the binding generation better, and also comes up with a better concept on the actual target directory of the generated files.

It seems like this was the only requested change. I'd suggest we merge this PR then.